jwst/stpipe/core.py

```diff
@@ -1,16 +1,17 @@
 """
 JWST-specific Step and Pipeline base classes.
 """
+import logging
```

Comment:
not sure why this reordering happened.  Is it necessary?  Did I do this or was it the work of @braingram ?

Reply:
Git blame tells me it was https://github.com/spacetelescope/jwst/commit/98fb07f7406b713d47167c9f7284eb7a5f975452
I'm not sure if this was done by isort, ruff, etc due to other changes in the file. Reverting the changes or keeping them is fine with me.

Reply:
might as well just leave them